### PR TITLE
Scripts: Remove deprecated `format-js` command

### PR DESCRIPTION
**Pull request by gziolo**
## What?

`format-js` command from `@wordpress/scripts` got deprecated and replaced with `format` in #30240. It's about time to remove it completely as it's been almost a year since it happened. It was undocumented all that time so the old projects would already notice the warning.

## Why?

It's no longer used.

## How?

Removes the code.

### CODE REVIEW

**Review by ockham — CHANGES_REQUESTED** [review]
Some more occurrences to remove:

https://github.com/WordPress/gutenberg/blob/47a535914907153c62086f0d7bea521c628eca53/package.json#L258

https://github.com/WordPress/gutenberg/blob/47a535914907153c62086f0d7bea521c628eca53/docs/how-to-guides/data-basics/1-data-basics-setup.md?plain=1#L161

File: docs/how-to-guides/data-basics/1-data-basics-setup.md [hunk 1]
```diff
@@ -8,33 +8,39 @@ Also, this tutorial will lean heavily on Redux concepts such as state, actions,
 
 We'll do all the development inside of a WordPress plugin. Let's start by creating a `wp-content/plugins/my-first-gutenberg-app` directory in your local WordPress environment. We will need to create four files inside that directory:
 
-* my-first-gutenberg-app.php – to create a new admin page
-* src/index.js – for our JavaScript application
-* style.css – for the minimal stylesheet
-* package.json – for the build process
+-   my-first-gutenberg-app.php – to create a new admin page
```
Comment (gziolo) [inline docs/how-to-guides/data-basics/1-data-basics-setup.md:11]: I have Prettier integrated with my IDE and this is why these changes got included. It's also surprising because all Markdown files were formatted with Prettier a few months ago...

Comment (ockham) [reply docs/how-to-guides/data-basics/1-data-basics-setup.md:11]: Yeah, that's kinda weird 🤔  I hope that we don't end up accidentally using different configs for prettier/markdown in our scripts and in our editor configs.

Anyway, it's just whitespace 😬  Even if this wasn't the format we want, it wouldn't be the end of the world.

**Review by ockham — APPROVED** [review]
LGTM! Thanks @gziolo! :shipit:

**Comment by adamziel** [comment]
This was backported to wp/6.0 branch to be included in WordPress 6.0 Beta 3 tomorrow: 73e2443ca935c27551a367728c0dc8223a25db0d
